doc/examples/segment_quickbundles.py

```diff
@@ -28,9 +28,10 @@
 Load fornix streamlines.
 """
 
-streams, hdr = tv.read(fname)
+fornix = load_tractogram(fname, 'same',
+                         bbox_valid_check=False).streamlines
 
-streamlines = [i[0] for i in streams]
+streamlines = Streamlines(fornix)
```

Comment:
Shouldn't this be streamlines = fornix?

Reply:
Or the load_tractogram().streamlines directly

Reply:
This is a tutorial, I just wanted to be explicit for the user. So now, it is:
`streamlines = fornix.streamlines`